ci/builders/mac_unopt.json

```diff
@@ -111,6 +111,109 @@
                     "type": "local"
                 }
 
+            ]
+        },
+        {
+            "archives": [
+                {
+                    "base_path": "out/host_debug_unopt_arm64/zip_archives/",
+                    "type": "gcs",
+                    "include_paths": [
+                    ],
+                    "name": "host_debug_unopt_arm64"
+                }
+            ],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Mac-12",
+                "cpu=arm64"
+            ],
+            "gclient_custom_vars": {
```

Comment:
gclient_variables?

Reply:
This was copy-pasted from the other ones.